plugins/ufm_consumer_plugin/.ci/ci_matrix.yaml

```diff
@@ -0,0 +1,54 @@
+---
+job: ufm_consumer-plugin
```

Comment:
can you tell what is the UFM Consumer plugin image size?

Reply:
1288595590 - 1.3 G

Reply:
new image size? based on UFM container....

Reply:
913M